### DISCUSSION TITLE
Welcome to openethereum Discussions!

### DISCUSSION THREAD

**Discussion by igorbarinov** [openethereum/openethereum#655] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KishanV** [comment]
Thanks for opening Discussions @igorbarinov. 

Can you please tell how many nodes are using openethereum?

**Reply by igorbarinov** [reply]
You are welcome. 
Based on Ethernodes https://ethernodes.org/ OpenEthereum is a client number three by the number of nodes.

The client was discontinued by original creators in December 2019 https://www.parity.io/blog/parity-ethereum-openethereum-dao/ and later by Gnosis https://medium.com/openethereum/gnosis-joins-erigon-formerly-turbo-geth-to-release-next-gen-ethereum-client-c6708dd06dd on June 2, 2021.

For now, the ex-xDai team is supporting OpenEthereum and working on TheMerge https://github.com/orgs/openethereum/projects/3

What's your interest in the client?
